content/doc/tutorials/create-a-pipeline-in-blue-ocean.adoc

```diff
@@ -8,430 +8,297 @@ section: doc
+. Back in Blue Ocean, paste the access token into the *Your GitHub access token* field, and select *Connect*.
++
 [.boxshadow]
-image:tutorials/pipeline-in-blueocean-01-connecting-to-github.png[alt="Connecting
-to GitHub",width=80%] +
-  [[choose-github-account]]Jenkins now has access to your GitHub account
-  (provided by your access token).
-. In *Which organization does the repository belong
-  to?*, click your GitHub account (where you forked the repository
-  <<fork-sample-repository,above>>).
-. In *Choose a repository*, click your forked repository
-  *creating-a-pipeline-in-blue-ocean*.
-. Click *Create Pipeline*. +
-  Blue Ocean detects that there is no Jenkinsfile at the root level of the
-  repository's `master` branch and proceed to help you create one. (Therefore,
-  you'll need to click another *Create Pipeline* at the end of the page to
-  proceed.) +
-  *Note:* Under the hood, a Pipeline project created through Blue Ocean is
-  actually "multibranch Pipeline". Therefore, Jenkins looks for the presence of
-  at least one Jenkinsfile in any branch of your repository.
+image:tutorials/pipeline-in-blueocean-01-connecting-to-github.png[alt="Connecting to GitHub",width=80%]
++
+[[choose-github-account]]Jenkins now has access to your GitHub account, using your access token.
+. In *Which organization does the repository belong to?*, select your GitHub account where you forked the repository <<fork-sample-repository,above>>.
+. In *Choose a repository*, select your forked repository `creating-a-pipeline-in-blue-ocean`.
+. Select *Create Pipeline*.
+* Blue Ocean detects that there is no Jenkinsfile at the root level of the repository's `master` branch and helps you create one.
+You must select *Create Pipeline* again at the end of the page to proceed.
 
+TIP: Under the hood, a Pipeline project created through Blue Ocean is actually a "multibranch Pipeline".
+Therefore, Jenkins looks for the presence of at least one Jenkinsfile in any branch of your repository.
 
 === Create your initial Pipeline
 
-. Following on from creating your Pipeline project
-  (<<create-your-pipeline-project-in-blue-ocean,above>>), in the Pipeline
-  editor, select *docker* from the *Agent* dropdown in the *Pipeline Settings*
-  panel on the right. +
+. Following on from creating your Pipeline project <<create-your-pipeline-project-in-blue-ocean,above>>, in the Pipeline editor, select *Docker* from the *Agent* dropdown in the *Pipeline Settings* panel on the right side of the page.
++
 [.boxshadow]
-image:tutorials/pipeline-in-blueocean-11-initial-pipeline-editor-page.png[alt="Initial
-to GitHub",width=100%]
-. In the *Image* and *Args* fields that appear, specify `node:6-alpine` and
-  `-p 3000:3000` respectively. +
+image:tutorials/pipeline-in-blueocean-11-initial-pipeline-editor-page.png[alt="Initial to GitHub",width=100%]
+. In the *Image* and *Args* fields that appear, specify `node:6-alpine` and `-p 3000:3000` respectively.
```

Comment:
Could you try changing the node version to a newer version.  Does it still work?
```suggestion
. In the *Image* and *Args* fields that appear, specify `node:lts-alpine` and `-p 3000:3000` respectively.
```

Reply:
@kmartens27 I realized that there is no compelling reason to delay the merge of this for my question about Alpine version.  Your new phrasing is a well-reviewed and well done improvement.

If you find that the newer version of Node.JS works as well, can you submit that change as a separate pull request?